package/network/utils/wwan/files/wwan.sh

```diff
@@ -108,11 +108,11 @@ proto_wwan_teardown() {
 	dat_device=$(uci_get_state network $interface dat_device)
 
 	case $driver in
-	qmi_wwan)	proto_qmi_teardown $@ ;;
-	cdc_mbim)	proto_mbim_teardown $@ ;;
-	sierra_net)	proto_mbim_teardown $@ ;;
-	comgt)		proto_3g_teardown $@ ;;
-	*cdc_ncm)	proto_ncm_teardown $@ ;;
+	qmi_wwan)		proto_qmi_teardown $@ ;;
```

Comment:
with the current change, if I keep the original tab, the last line (cdc_ether|*cdc_ncm) will be not alligned.

Are you sure that this is the best practice?

Bye.